### PR TITLE
[RU]Newbie cloudflareClient (fix 403)

### PR DESCRIPTION
**Pull request by e-shl**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/ru/newbie/src/eu/kanade/tachiyomi/extension/ru/newbie/Newbie.kt [hunk 1]
```diff
@@ -66,8 +68,10 @@ class Newbie : ConfigurableSource, HttpSource() {
 
     private var branches = mutableMapOf<String, List<BranchesDto>>()
 
+    private val userAgentRandomizer = "${Random.nextInt().absoluteValue}"
+
     override fun headersBuilder(): Headers.Builder = Headers.Builder()
-        .add("User-Agent", "Tachiyomi " + System.getProperty("http.agent"))
+        .add("User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36 Edg/100.0.$userAgentRandomizer")
```
Comment (alessandrojean) [inline src/ru/newbie/src/eu/kanade/tachiyomi/extension/ru/newbie/Newbie.kt:74]: Isn't it better to just not set it and let users change through the new app setting?

Comment (e-shl) [reply src/ru/newbie/src/eu/kanade/tachiyomi/extension/ru/newbie/Newbie.kt:74]: > Isn't it better to just not set it and let users change through the new app setting?

it's easier

Comment (alessandrojean) [reply src/ru/newbie/src/eu/kanade/tachiyomi/extension/ru/newbie/Newbie.kt:74]: Adding a explicit user-agent (even with the random part) will allow the site owners to easily block on the Cloudflare panel though. Specially if they already have enabled the protection, it's very likely they will do that.

Not setting will make the extension use the value from the app preferences so users can change to whatever user agent string they want and the site owners can't block a specific one.

Comment (e-shl) [reply src/ru/newbie/src/eu/kanade/tachiyomi/extension/ru/newbie/Newbie.kt:74]: They do not specifically block the tachiemi user-agent, it just started blocking the client for some, that is, it is necessary to periodically enter a captcha. I'm changing user-agent for a working bundle as on remanga.

Comment (alessandrojean) [reply src/ru/newbie/src/eu/kanade/tachiyomi/extension/ru/newbie/Newbie.kt:74]: Fine, sure... but if they start blocking in the future, it's better to remove it next time and let the users choose one on the app.

**Pull request merged**